core/src/main/java/tech/tablesaw/api/BooleanColumn.java

```diff
@@ -667,15 +671,55 @@ public boolean hasNext() {
          * @throws java.util.NoSuchElementException if the iteration has no more elements
          */
         @Override
-        public Boolean next() {
+        public boolean nextBoolean() {
             byte b = iterator.nextByte();
             if (b == (byte) 0) {
                 return false;
             }
             if (b == (byte) 1) {
                 return true;
             }
-            return null;
-        }
-    }
+            return false;
+        }
+    }
+    
+    // fillWith methods
+    
+	@Override
+	public BooleanColumn fillWith(BooleanIterator iterator) {
```

Comment:
you have tabs here and spaces above. should be consistent with the rest of tablesaw